pyccel/parser/semantic.py

```diff
@@ -4799,3 +4802,50 @@ def _build_CmathPhase(self, func_call):
         else:
             self.insert_import('math', AsName(MathAtan2, 'atan2'))
             return MathAtan2(PythonImag(var), PythonReal(var))
+
+    def _visit_SetUpdate(self, expr):
+        """
+        Method to navigate the syntactic DottedName node of an `update()` call.
+
+        The purpose of this `_visit` method is to construct new nodes from a syntactic 
```

Comment:
I suspect that this doesn't work. Please check #1844, especially the new docs and check if this method has the right name. If it does have the right name please move it out of this section of the file which groups:
https://github.com/pyccel/pyccel/blob/efce97ed3b1a04d8b89a0876663a7b8a8d74a87f/pyccel/parser/semantic.py#L4512-L4514

and into the section for `_visit` functions

Reply:
Fixed : 9a9cdde